agent/js/src/wd_server.js

```diff
@@ -896,12 +896,20 @@ WebDriverServer.prototype.clearPageAndStartVideoDevTools_ = function() {
       this.networkCommand_('setExtraHTTPHeaders', {'headers': this.task_.headers});
     this.pageCommand_('enable');
   }
+};
+
+/**
+ * Blanks out the browser at the beginning of a test, using DevTools.
+ * @private
+ */
+WebDriverServer.prototype.holdBlankAndStartVideoDevTools_ = function() {

```

Comment:
Other name ideas welcome, just wanted to update from what I assume was drift over time.